web/app.js

```diff
@@ -2636,6 +2639,7 @@ function webViewerWheel(evt) {
   }
 
   if (
+    (evt.ctrlKey && !PDFViewerApplication._isCtrlKeyDown) /* Pinch-to-zoom */ ||
```

Comment:
Unfortunately I don't, at all, understand this patch :-(

How can the <kbd>Ctrl</kbd> key be active in the wheel-event here, and simultaneously not be active at the document level?
If this works generally correct (I don't have a Mac to test with), then I'd expect a much larger and more detailed comment (placed just above rather than inline) that *clearly* explains exactly what's going on here.

(As-is this seems like the thing that won't make sense even to you, as the patch author, in e.g. a months time.)

Reply:
Also, will this *only* target MacOS or could it lead to issues/inconsistencies on Linux/Windows?

Reply:
Sorry I thought it was something known (entry `Pinch-to-zoom maps to WheelEvent + ctrl key`):
https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent#browser_compatibility

I don't think it could lead to some inconsistencies but if you can imagine one please tell me.

Reply:
Ah, so that was the missing piece of context!

In that case, as already mentioned, please add a more detailed comment that mentions this to help future readers of the code.